modules/rhel-compute-updating.adoc

```diff
@@ -69,10 +67,10 @@ By default, the base OS RHEL with "Minimal" installation option enables firewall
 +
 [source,terminal]
 ----
-# subscription-manager repos --disable=rhel-7-server-ose-4.8-rpms \
-                             --enable=rhel-7-server-ose-4.9-rpms  \
-                             --enable=rhel-7-fast-datapath-rpms   \
-                             --enable=rhel-7-server-optional-rpms
+# subscription-manager repos --disable=rhel-8-server-ose-4.9-rpms \
```

Comment:
I think we only need to update ocp related repo here.

In current 4.9 doc - https://docs.openshift.com/container-platform/4.9/machine_management/adding-rhel-compute.html#rhel-preparing-node_adding-rhel-compute, we already clarified the following repos need to be enabled.

```
For RHEL 8 nodes, you must enable the following repositories:

# subscription-manager repos \
    --enable="rhel-8-for-x86_64-baseos-rpms" \
    --enable="rhel-8-for-x86_64-appstream-rpms" \
    --enable="rhocp-4.9-for-rhel-8-x86_64-rpms" \
    --enable="fast-datapath-for-rhel-8-x86_64-rpms"
```

When upgrading the compute node to OCP-4.10, we should only need to 
```
# subscription-manager repos --disable=rhocp-4.9-for-rhel-8-x86_64-rpms \
                             --enable=rhocp-4.10-for-rhel-8-x86_64-rpms  
```